scripts/suite.py

```diff
@@ -168,6 +168,7 @@
                               with --rerun argument. This number can be found
                               in the output of teuthology-suite command. -1
                               for a random seed [default: -1].
+ --skip                       Skip the priority check.
```

Comment:
The `--skip` option is ambiguous. It is better to rename it to `--check-priority` with defaults to False.

Reply:
I think the idea is to stop inappropriate use of high priority by default, so we'd want the check on by default. maybe ``--force-priority`` or something?

Reply:
The `--force-priority` which allows to skip the check looks fine to me.

Reply:
@kshtsk Thank you for the detailed explanation.  :)